package.json

```diff
@@ -11,7 +11,7 @@
     "type": "module",
     "path": "dist/index.js",
     "source": "src/index.ts",
-    "host": "^9.0.0-rc.95",
```

Comment:
This would mean the addon doesn't work anymore on old directus installation, which (as far as I know), it still does.

Reply:
reverted it back to support v9.0, I just didn't test on it.